01week/datatypes.js

```diff
@@ -0,0 +1,46 @@
+//display the current day and time
+const today = new Date();
+const dd = today.getDate();
+const mm = today.getMonth();
+const yyyy = today.getFullYear();
+Date(dd,mm,yyyy)
+
+//convert a number to a string
+var num = 15;
+var n = num.toString();
+console.log(num);
+
+//string to the number
+Number("5");
```

Comment:
This is a variable. I need a function that converts ANY string into a number and converts it.